.github/workflows/root-ci.yml

```
@@ -8,11 +8,16 @@ on:
     paths-ignore:
       - 'doc/**'
       - 'documentation/**'
+      - 'README/ReleaseNotes/**'
```

```suggestion
      - 'README/**'
```
i.e. is there anything under `README` that should trigger a build?

---

I had the doubt myself.... The list of contributors perhaps?